packages/supabase/lib/src/supabase_client.dart

```diff
@@ -125,49 +120,47 @@ class SupabaseClient {
   SupabaseClient(
     this.supabaseUrl,
     this.supabaseKey, {
-    String? schema,
-    bool autoRefreshToken = true,
+    PostgrestClientOptions postgrestOptions = const PostgrestClientOptions(),
+    GoTrueClientOptions goTrueOptions = const GoTrueClientOptions(),
+    StorageClientOptions storageOptions = const StorageClientOptions(),
+    RealtimeClientOptions realtimeClientOptions = const RealtimeClientOptions(),
```

Comment:
For the naming conventions, could we rename `postgrest` to `db` and `gotrue` to `auth` so it's a bit more aligned with the js client?
https://github.com/supabase/supabase-js/blob/master/src/lib/types.ts

Reply:
So it's `DBClientOptions`? I feel like that's not really aligned to any other naming, because `db` is currently nowhere used. Or do you only mean the argument name and keep the class name?

Reply:
Hmm, yeah js naming convention might be a bit misleading or ambiguous. Let's stick to what you have here for Postgrest then. I think auth can be updated though.

Reply:
Agree, I've updated it now to `AuthClientOptions`